src/Renderer/Device/VulkanDevice.h

```diff
@@ -0,0 +1,95 @@
+#pragma once
+
+#include "../Renderer.h"
+#include "../../Window/Window.h"
+
+#include "Utils/DebugUtilsMessenger.h"
+#include "Utils/Extensions.h"
+#include "Utils/QueueFamilyIndices.h"
+#include "Utils/PhysicalDevice.h"
+#include "Utils/SwapChainSupportDetails.h"
+
+// std lib headers
+#include <string>
+#include <vector>
+#include <array>
+
+namespace SnekVk {
+
+class VulkanDevice {
+	public:
+
+#ifdef DEBUG
+	const bool enableValidationLayers = true;
+#else
+    const bool enableValidationLayers = false;
+#endif
```

Comment:
```suggestion
#define DEBUG
	ENABLE_VALIDATION_LAYERS true
#else
        ENABLE_VALIDATION_LAYERS false
#endif

const bool enableValidationLayers = ENABLE_VALIDATION_LAYERS;
```

Consider moving macro switch out of the class with something like this - might be able to simply define with `DEBUG`.

Reply:
Changed.